routes/flag.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const authenticate = require("../middlewares/authenticate");
+const { authorizeUser } = require("../middlewares/authorization");
+const flagController = require("../controllers/flag");
+const { addFeatureFlag } = require("../middlewares/validators/flag");
+const { SUPER_USER } = require("../constants/roles");
+
+router.post("/add", authenticate, authorizeUser(SUPER_USER), addFeatureFlag, flagController.addFlag);
```

Comment:
HTTP paths may be better off with just nouns and no verbs. Here is a short read from [Google Cloud Blog](https://cloud.google.com/blog/products/api-management/restful-api-design-nouns-are-good-verbs-are-bad).

What are your views in having a POST to `/feature-flag`, instead of `/flag/add`?